src/CashuWallet.ts

```diff
@@ -206,13 +200,21 @@ class CashuWallet {
 		return { payload, amount1BlindedMessages, amount2BlindedMessages };
 	}
 	//keep amount 1 send amount 2
-	private splitReceive(amount: number, amountAvailable: number) {
+	private splitReceive(
+		amount: number,
+		amountAvailable: number
+	): { amount1: number; amount2: number } {
 		const amount1: number = amountAvailable - amount;
 		const amount2: number = amount;
 		return { amount1, amount2 };
 	}
 
-	private async createRandomBlindedMessages(amount: number) {
+	private async createRandomBlindedMessages(amount: number): Promise<{
+		blindedMessages: Array<SerializedBlindedMessage>;
+		secrets: Array<Uint8Array>;
+		rs: Array<bigint>;
+		amounts: Array<number>;
```

Comment:
maybe type worthy? bellow in createBlankOutputs we have the same object with only `amounts` missing